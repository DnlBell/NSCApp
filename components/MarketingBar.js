import React, {Component} from 'react';
import { View, Text, TouchableHighlight, StyleSheet } from 'react-native';

class MarketingBar extends Component {

    render() {
        return(
            <View style={styles.container}>
                <Text style={styles.marketingText}>Offer your own classes on Mindspand!</Text>
                <TouchableHighlight style={styles.marketingButton}>
                    <Text>Learn more</Text>
                </TouchableHighlight>
            </View>
        )
    }
}

const styles = StyleSheet.create({
    container: {
        flex:1,
        backgroundColor: '#FF6622',
        alignItems: 'center'
    },
    marketingText:{
        color:'#fff',
        fontSize: 18,
        margin: 8
    },
    marketingButton: {
        flex:1,
        alignItems: 'center',
        backgroundColor: '#fff',
        marginBottom: 18,
        marginLeft: 12,
        marginRight: 12,
        padding: 18,
        width:200
    }
})

export default MarketingBar