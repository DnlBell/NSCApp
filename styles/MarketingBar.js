import {StyleSheet} from 'react-native';
import {Colors, Margins, FontSizes} from './Common'

export default StyleSheet.create({
    container: {
        flex:1,
        backgroundColor: Colors.AccentColorLight,
        alignItems: 'center'
    },
    marketingText:{
        color: Colors.PrimaryColor,
        fontSize: FontSizes.ReadingFontSize,
        margin: 8
    },
    marketingButton: {
        flex:1,
        alignItems: 'center',
        backgroundColor: Colors.PrimaryColor,
        marginBottom: Margins.DefaultMargin,
        padding: Margins.DefaultMargin,
        width:200
    }
});