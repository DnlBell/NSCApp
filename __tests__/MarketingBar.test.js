import MarketingBar from '../components/MarketingBar';
import renderer from 'react-test-renderer';
import React from 'react';

it('renders correctly', () => {
  const tree = renderer
    .create(<MarketingBar />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});