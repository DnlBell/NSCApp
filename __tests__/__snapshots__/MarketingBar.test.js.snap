// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#FF6622",
      "flex": 1,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#FFF",
        "fontSize": 16,
        "margin": 8,
      }
    }
  >
    Offer your own classes on Mindspand!
  </Text>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#FFF",
        "flex": 1,
        "marginBottom": 18,
        "opacity": 1,
        "padding": 18,
        "width": 200,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
    >
      Learn more
    </Text>
  </View>
</View>
`;
